stan/math/prim/mat/prob/bernoulli_logit_glm_lpmf.hpp

```diff
@@ -95,9 +98,9 @@ typename return_type<T_x, T_alpha, T_beta>::type bernoulli_logit_glm_lpmf(
       beta_dbl[m] = value_of(beta_vec[m]);
     }
   }
+  const T_alpha_val &alpha_val = value_of(alpha);
```

Comment:
This makes me think that `T_alpha_val` should really be an `Eigen::Matrix` instead of `Eigen::Array`. As far as I know, we have no uses of `Eigen::Array` from the Stan library.

Reply:
I'm assuming that when it's multivariate, `value_of(alpha)` will return a `Eigen::Matrix`... is there any cost to converting that to an `Eigen::Array`?